apps/server/src/modules/authorization/authorization-reference.module.ts

```diff
@@ -0,0 +1,41 @@
+import { forwardRef, Module } from '@nestjs/common';
+import {
+	CourseGroupRepo,
+	CourseRepo,
+	LessonRepo,
+	SchoolExternalToolRepo,
+	LegacySchoolRepo,
+	SubmissionRepo,
+	TaskRepo,
+	TeamsRepo,
+	UserRepo,
+} from '@shared/repo';
+import { ToolModule } from '@src/modules/tool';
+import { LoggerModule } from '@src/core/logger';
+import { BoardModule } from '@src/modules/board';
+import { ReferenceLoader, AuthorizationReferenceService, AuthorizationHelper } from './domain';
+import { AuthorizationModule } from './authorization.module';
+
+/**
+ * Should only be used inside of the authorization module
```

Comment:
Ich fänds gut, das noch etwas auszuführen. Vorschlag (wobei ich mir ganz sicher bin, ob das richtig ist):
"This module is part of an intermediate state. In the future it should be replaced by an AuthorizationApiModule.
For now it is used where the authorization itself needs to load data from the database. 
Try to avoid using this module and load the needed data in your use cases and then use the normal AuthorizationModule!"

Reply:
Ich habe den Vorschlag rein kopiert. Nur das try raus geworfen. Damit niemand in Versuchung geführt wird, ohne Absprache.